### ISSUE TITLE
CrunchyScan (Madara)

### ISSUE THREAD

**Issue by pixelroll** [tachiyomiorg/tachiyomi-extensions#8336]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### Source name

CrunchyScan (Madara)

### Source link

https://crunchyscan.fr/

### Language

French

### Other details

This is the website of a small French team, which translates the least advanced webtoons in France.
For example  The Beginning After The End 
In short, all webtoons available on this site get a very fast and high quality translation.
Thanks

**Comment by pixelroll** [comment]
their discords currently has over 2000 members

**Comment by Arraiment** [comment]
The site uses captcha before every chapter so it might be difficult to implement

**Comment by Mitia99** [comment]
I hope that someone could work on it cause this site as many french exclusives trad.

**Comment by vetleledaal** [comment]
This is unlikely to be added due to the captcha. The site uses reCaptcha to get chapter pages, and it's actually checked server-side -- can't reuse old challenge responses, sending an array or setting it to null doesn't return the pages either.

If you somehow manage to bypass this the pages are in the `canvas_urls` array, base64 encoded. `dateFormat` is `SimpleDateFormat("d MMM yyy", Locale("fr")`

**Comment by alessandrojean** [comment]
> This is unlikely to be added due to the captcha. The site uses reCaptcha to get chapter pages, and it's actually checked server-side -- can't reuse old challenge responses, sending an array or setting it to null doesn't return the pages either.

A WebView interceptor can be used to bypass this, like how it was used in TOPTOON+ before its removal.
